terraform/mesh_aws/s3_logs.tf

```diff
@@ -1,16 +1,6 @@
 #tfsec:ignore:aws-cloudtrail-require-bucket-access-logging tfsec:ignore:aws-s3-enable-versioning
 resource "aws_s3_bucket" "s3logs" {
   bucket = "${local.name}-s3logs"
-  acl    = "log-delivery-write"
```

Comment:
is this a problem that we're removing the log delivery-write  ACL here ?  should we be adding a aws_s3_bucket_acl resource as well ?